RecoPixelVertexing/PixelVertexFinding/src/PixelVertexProducer.cc

```diff
@@ -41,7 +41,7 @@ PixelVertexProducer::PixelVertexProducer(const edm::ParameterSet& conf)
     track_pt_min   = PVcomparerPSet.getParameter<double>("track_pt_min");    
     if (track_pt_min != ptMin_) {
       if (track_pt_min < ptMin_)
-	edm::LogWarning("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
+	edm::LogInfo("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
```

Comment:
because nobody complained/noticed it earlier, and the fix is "fix the config".
It is not "years" it is at most "1 year", in fact less given the the config change was integrated (checking JIRA) on Nov 5th 2014